go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/google/go-github/v26
 require (
 	github.com/golang/protobuf v1.2.0 // indirect
 	github.com/google/go-querystring v1.0.0
+	github.com/jchavannes/go-pgp v0.0.0-20170517064339-cb4b0b0ac8cf
```

Comment:
Why is this line needed if we are only importing `golang.org/x/crypto/openpgp`?
Can you please revert the changes to `go.mod` and `go.sum` if these changes aren't needed?

Reply:
Done! They were there with my first implementation (had nice sugar functions) but I decided to stick with `golang.org/x/crypto/openpgp` and stay away from the sugar.